### DISCUSSION TITLE
PocketStore.io

### DISCUSSION THREAD

**Discussion by jonathan-martz** [pocketbase/pocketbase#5865] [Show and tell]
Iam currently working on a Ecommerce Store which is based on a **PocketBase** instance.

My Name would be **PocketStore** under **PocketStore.io**.

It will be **Open Source**.

**Progress:**
- Checkout is working realy well, with few Payment Methods like Klarna, PayPal and Stripe (only dynamic config for these are missing)
- Checkout Order Flow is already working
- Translation is implemented, currently adding away day a bunch of translations, currently German and English.
-  Docs is the next big Project **WIP**
-  Customer Account is still missing
-  Product Page is work in Progress
-  Categories is work in Progress
-  Customer Support vie Realtime Api is planned.

**Comment by benallfree** [comment]
I am so excited to see this! Do you have a github URL yet?

I'd love to collab

**Reply by jonathan-martz** [reply]
I will send you a Link via Email.

Tagged the first version, working on Customization for Stores, on the Docs.

**Reply by VictorioBerra** [reply]
How's the progress? Anything live yet?

**Comment by jonathan-martz** [comment]
Still working on it.

Plan is to cleanup everything and the release version 0.0.6 as a kind of Beta version end of August.

Until 0.0.6 there will be not that much changes.
- Cleanup Docs
- do some hooks for Order Emails (currently learning much more go)
- Work on administration UI for SuperAdmins (Translations,Checkout,Product Catalog)
- finish Demo Stops with custom Interfaces (3d-world.pro,nerdythings.shop)
- Catalog has to be finished

Docs: (second version still WIP)
https://github.com/pocketstore-io/docs-next

https://www.pocketstore.io/

Demo Stores: (Plain Storefront)
https://github.com/pocketstore-io/demo
https://github.com/pocketstore-io/storefront

https://demo.pocketstore.io/de/

Forks are extended version with override components from storefront.
The Goal is to show custom storefronts.

https://ikea.pocketstore.io/de
https://www.gummersbach-biker.shop/de
https://www.my-media.world/de

There are a few more coming, as you can see in forks from demo repo.

**Comment by jonathan-martz** [comment]
### Small Status update whats the Plan for 0.0.6 is.

More or less much things done, button iam curently moving in real life, so things are little delayed.
Plan for 0.0.6 still same Goals, i prepared some demo stores.

Still todos:
- Catalog has to be finished
- Cleanup Docs
- Hooks missing for Orders in go

New Improvments planned for 0.0.6:
- Plugin structure similiar to current custom extensions

**Comment by jonathan-martz** [comment]
So for me the Release Plan changed a little bit, will tag 0.0.6 after testing some major changes for Plugins with its own Store.
I have to develop Themes for the Stores with Header,Footer and maybe diasuyui.css config.
https://plugins.pocketstore.io/de

Plugins need do be created with content from Storefront or Baseline as it is called in Demo Repo.

After that i will continue for a few month to finish Docs and also Storefront. Then i have to update Demo Stores.
Storefront should be done in 0.0.7
Dont know what should be in 0.0.8 to finish the Storefront, but i will see.
Docs should be 0.0.9

**Comment by antonyryan** [comment]
I will help you translate it to portuguese-BR.

**Reply by jonathan-martz** [reply]
Thanks for your Offer but currently the Project is on Work in Progress.
Currently only german and english is needed.
At the moment iam working on v0.0.6 with a rewritten Plugin System, including Translation.

Maybe at v0.1.0 with first beta version we could need some translation.

I already updated the Milestones with Issue for 0.0.6 and 0.0.7
https://github.com/pocketstore-io/demo

**Reply by antonyryan** [reply]
Feel free to reach if you need the translation. I would be happy to help.

**Comment by jonathan-martz** [comment]
Version 0.0.6 was released an 0.0.7 is planned for end of this Month.
https://github.com/pocketstore-io/demo/milestone/2

**Comment by jonathan-martz** [comment]
**0.0.7 - Release Notes**

- LanguageSwitcher from the Demo Store
- dozent plugins for better maintainability
- schema.go for automated schema for PocketBase
- checks.go for automated plugin checks, such as translation page length and component length
- improved requirements checks for plugins
- improved product view with new template

Install.sh is currently in Construction ...

Working also on Docs -> www.pocketstore.io
